projects/demo/src/app/app.component.spec.ts

```diff
@@ -43,13 +43,13 @@ describe('AppComponent', () => {
   });
 
   it('should enable the submit button when filling the first name', () => {
-    tester.firstName.fillWith('John');
+    void tester.firstName.fillWith('John');
     expect(tester.submit.disabled).toBe(false);
   });
 
   it('should display the greeting when submitting the form', () => {
-    tester.firstName.fillWith('John');
-    tester.submit.click();
+    void tester.firstName.fillWith('John');
+    void tester.submit.click();
```

Comment:
OOC why not call await instead?

Reply:
Because this demonstrates the imperative mode, where we don't need to await anything. But the linter complains about promises not being awaited.